pkg/composer/serviceparser/serviceparser_test.go

```diff
@@ -348,6 +348,10 @@ services:
   foo:
     image: nginx:alpine
     network_mode: host
+    container_name: nginx
+  bar:
+    image: alpine
+    network_mode: container:nginx
 `
```

Comment:
```suggestion
	const dockerComposeYAML = fmt.Sprintf(`
services:
  foo:
    image: %s
    network_mode: host
    container_name: nginx
  bar:
    image: %s
    network_mode: container:nginx
`, testutil.NginxAlpineImage,  testutil.AlpineImage)
```

Reply:
fixed

Reply:
Hi @fahedouch, I have to change it back. `testutil.NginxAlpineImage,  testutil.AlpineImage` are defined in `testutil_linux.go` which will fail the FreeBSD test. Not sure if it can be moved to the common `testutil.go`. So I will use `alpine:3.14` as other tests in the file.

https://github.com/containerd/nerdctl/blob/a04a4e5645065b91a063cfb2a8de46f56cde88b3/pkg/testutil/testutil_linux.go.